src/browser/modules/Sidebar/Documents.tsx

```diff
@@ -41,8 +41,8 @@ import {
   StyledHeaderContainer
 } from './styled'
 
-export const formatDocVersion = (v = ''): string => {
-  if (!semver.valid(v)) {
+export const formatDocVersion = (v: string | null = ''): string => {
```

Comment:
`v` is probably `undefined` rather than `null`? Please correct me if I'm wrong.

Reply:
It is actually null in the initialState in redux.